.github/CODEOWNERS

```diff
@@ -643,12 +643,6 @@
 # ServiceLabel: %Monitor - Exporter
 # ServiceOwners:                                                   @cijothomas @reyang @rajkumar-rangaraj @TimothyMothra @vishweshbankwar
 
-# PRLabel: %Monitor - LiveMetrics
```

Comment:
This should removed only after the deprication announcement of the Live Metrics beta package. We own the cleanup of this component.

Reply:
Please don't remove the Service Label part, even then.   There's enough history in the issues that the AI label bot will continue to use this unless we rename the label to whatever we would want to migrate to and then retrain the model.

Reply:
PR portions can be removed as you see fit.

Reply:
reverted this change